templates/alerts/tags/alerts.html

```diff
@@ -5,10 +5,10 @@
     <div class="container">
         <div class="row">
             <div class="col-md-12">
-                <span class="alert-message__label">{{ alert.title }}</span>
-                <span class="alert-message__description">{{ alert.message|richtext }}</span>
+                <span class="alert-message__label" data-alert-title="{{ alert.title }}">{{ alert.title }}</span>
+                <span class="alert-message__description" data-alert-message="{{ alert.message|richtext }}">{{ alert.message|richtext }}</span>
```

Comment:
Hi @JamesWChan - I've just been looking at the `richtext` filter within the Wagtail docs. https://docs.wagtail.io/en/stable/topics/writing_templates.html#rich-text-filter it says this filter will output HTML, which isn't going to work in the context of a `data-` attribute. I might be wrong about this but was wondering if we might be able to use the Django `striptags` filter so that we just get the content: https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#striptags

Reply:
Thanks @gtvj, I've changed that now.